app/src/main/java/io/github/jbellis/brokk/AnalyzerWrapper.java

```diff
@@ -468,6 +392,9 @@ private synchronized void refresh(Supplier<IAnalyzer> supplier) {
         logger.trace("Refreshing analyzer (full)");
         future = runner.submit("Refreshing code intelligence", () -> {
             try {
+                if (listener != null) {
+                    listener.beforeEachBuild();
+                }
```

Comment:
Also I brought back the spinner on Workspace when code intelligence is (re)building.